cmd/benchmark/deposit_transfer_scenario.go

```diff
@@ -0,0 +1,90 @@
+///
+// Copyright 2019 Insolar Technologies GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+///
+
+package main
+
+import (
+	"fmt"
+	"math/big"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/insolar/insolar/api/sdk"
+)
+
+type depositTransferScenario struct {
+	insSDK           *sdk.SDK
+	members          []sdk.Member
+	migrationDaemons []sdk.Member
+
+	balanceCheckMembers []sdk.Member
+}
+
+func (s *depositTransferScenario) canBeStarted() error {
+	if len(s.members) < concurrent {
+		return fmt.Errorf("not enough members for start")
+	}
+
+	return nil
+}
+
+func (s *depositTransferScenario) prepare(repetition int) {
+	members, err := getMembers(s.insSDK, concurrent, true)
+	check("Error while loading members: ", err)
+
+	if useMembersFromFile {
+		members = members[:len(members)-2]
+	}
+
+	s.members = members
+
+	s.migrationDaemons, err = s.insSDK.GetAndActivateMigrationDaemonMembers()
+	check("failed to get and activate migration daemons: ", err)
+
+	s.balanceCheckMembers = make([]sdk.Member, len(s.members), len(s.members)+2)
+	copy(s.balanceCheckMembers, s.members)
+	s.balanceCheckMembers = append(s.balanceCheckMembers, s.insSDK.GetFeeMember())
```

Comment:
add MigrationAdminMember to this balanceCheckMembers too:
`s.balanceCheckMembers = append(s.balanceCheckMembers, s.insSDK.GetMigrationAdminMember())`
we do this so we can use one file with saved entities in all scenario (check wallet_to_wallet_transfer_scenario.go and migration_scenario.go)